### PR TITLE
Start testing config v2 on our project

### PR DESCRIPTION
**Pull request by stsewd**
We'll need to activate the `ALLOW_V2_CONFIG_FILE` flag on the project

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 2
+formats: all
+sphinx:
+  configuration: docs/conf.py
+  fail_on_warning: true
+python:
+  install:
+    requirements: requirements.txt
```
Comment (stsewd) [inline .readthedocs.yml]: We need to change this later, when https://github.com/rtfd/readthedocs.org/pull/4740 is deployed

**Comment by ericholscher** [comment]
Flag is now set.

File: .readthedocs.yml (hunk 1 above)
Comment (ericholscher) [reply .readthedocs.yml]: I wonder if we should try and deploy this first.

Comment (ericholscher) [reply .readthedocs.yml]: Cool, lets ship it then :)

**Pull request merged**